### PR TITLE
Document the new features in version 2.7.0

### PR DESCRIPTION
**Pull request by settermjd**
This PR documents the new features in version 2.7.0. These are:

- New Features Wizard
- Pattern and Fingerprint Lock
- Upload directly from the camera
- GIF support

### Implements

https://github.com/owncloud/documentation/issues/4035.

### CODE REVIEW

**Comment by settermjd** [comment]
@davigonz & @michaelstingl, ready for your review and feedback.

File: user_manual/android_app.rst [hunk 1]
```diff
@@ -33,6 +33,26 @@ When you download your ownCloud Android App from the Google Play store, it will
 
 It will either update automatically or give you a notification that an upgrade is available. If you are using an ownCloud Android app from a custom repository, e.g., your employer, then it will update in accordance with their policies.
 
+New Features Wizard
+-------------------
+
+.. versionadded:: 2.7.0
```
Comment (davigonz) [inline user_manual/android_app.rst]: Should this line and the `ref` ones be shown like the image below?

screen shot 2018-05-04 at 11 59 38

File: user_manual/android_app.rst [hunk 2]
```diff
@@ -305,6 +341,26 @@ Tap it and enter the name of the new folder in the *Folder name* dialog.
 Then, tap the newly created folder and tap *Choose* in the bottom right-hand corner. 
 You'll see that the path has been updated.
 
+.. _upload_pictures_directly_from_the_camera_label:
+
+Upload Pictures Directly From The Camera
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. versionadded:: 2.7.0
```
Comment (davigonz) [inline user_manual/android_app.rst]: Same here

File: user_manual/android_app.rst [hunk 3]
```diff
@@ -320,6 +376,37 @@ The bottom section of the *"Settings"* screen has links to:
 - Feedback 
 - The version number
 
+.. _pattern_and_fingerprint_lock_label:
+
+Pattern Lock and Fingerprint Lock
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 2.7.0
```
Comment (davigonz) [inline user_manual/android_app.rst]: Same here

File: user_manual/android_app.rst [hunk 4]
```diff
@@ -320,6 +376,37 @@ The bottom section of the *"Settings"* screen has links to:
...
+
+In addition to the Passcode Lock and Pins, you also have the ability to use both a pattern and a fingerprint lock to protect access to your ownCloud app and its data.
```
Comment (davigonz) [inline user_manual/android_app.rst]: What do you mean with Pins here?

File: user_manual/android_app.rst [hunk 5]
```diff
@@ -305,6 +341,26 @@ Tap it and enter the name of the new folder in the *Folder name* dialog.
...
 You'll see that the path has been updated.
 
+.. _upload_pictures_directly_from_the_camera_label:
+
+Upload Pictures Directly From The Camera
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. versionadded:: 2.7.0
+
+.. figure:: images/camera/share-from-camera-owncloud-android-app.jpg
```
Comment (davigonz) [inline user_manual/android_app.rst]: I think that **Upload pictures directly from the camera** feature would fit better in **Files** section than in **Settings** one, below or even inside **Creating New Content** subsection

**Comment by davigonz** [comment]
@settermjd apart from the changes requested above, [https://doc.owncloud.org/android/](https://doc.owncloud.org/android/) is still showing Changes in 2.5.0 as the most recent version so it is left to include 2.6.0 and 2.7.0 changes. 

You can have a look at https://github.com/owncloud/android/releases to get the list of features included for both 2.6 and 2.7 versions.

**Comment by davigonz** [comment]
Any update on this @settermjd ?

File: user_manual/android_app.rst (hunk 1 above)
Comment (settermjd) [reply user_manual/android_app.rst]: No, they shouldn't. Not sure what's going on there.

Comment (settermjd) [reply user_manual/android_app.rst]: Oh, this is a screenshot from GitHub? If so, then that's to be expected, as it only has support for the reStructuredText format and not the sphinx-doc extensions. So, nothing to worry about.

File: user_manual/android_app.rst (hunk 4 above)
Comment (settermjd) [reply user_manual/android_app.rst]: That's a reference to the login pins. I'll update that line to make it more specific.

File: user_manual/android_app.rst (hunk 5 above)
Comment (settermjd) [reply user_manual/android_app.rst]: Ok.

**Comment by davigonz** [comment]
The changes look good to me, it seems there's some problems with Travis dependencies, is not related to our project but their environment, let's retry the tests.

**Pull request merged**